tests/gemmini/conv/test_gemmini_conv_no_pad.py

```diff
@@ -188,7 +188,7 @@ def test_conv_3():
     conv = conv.fission_after('for ocol_o in _:_ #0')
     conv = conv.reorder('orow', 'ocol_o')
     conv = conv.split('orow', 28, ['orow_o', 'orow_i'], perfect=True)
-    conv = conv.expand_dim('i_s: i8[_]', '30', 'krow + orow_i')
+    conv = conv.expand_dim('i_s: i8[_]', '30', 'krow + orow_i', unsafe_disable_checks=True)
```

Comment:
Where are the `FIXME` comments? Has an issue been opened?

Reply:
Ah forgot that